packages/static-module-record/test/fixtures/large.js

```diff
@@ -366,7 +366,7 @@ function makeModulePlugins(options) {
           if (!specs) {
             return;
           }
-          specs.forEach(spec => {
+          for (const spec of (specs)) {
```

Comment:
seemingly superfluous paren artifact
```suggestion
          for (const spec of specs) {
```